app/javascript/topics/TopicsShow__Root.re

```diff
@@ -0,0 +1,359 @@
+                      value={state.topicTitle}
+                      className="appearance-none block w-full bg-white text-gray-900 font-semibold border border-gray-400 rounded py-3 px-4 mb-2 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+                      type_="text"
+                    />
+                    <div className="flex justify-end">
+                      <button
+                        onClick={_ => send(ShowTopicEditor(false))}
+                        className="btn btn-subtle btn-small mr-2">
+                        {"Cancel" |> str}
+                      </button>
+                      <button
+                        onClick={updateTopic(state, send)}
+                        disabled={state.topicTitle |> Js.String.trim == ""}
+                        className="btn btn-primary btn-small">
+                        {"Update Topic" |> str}
+                      </button>
+                    </div>
+                  </div>
+                </DisablingCover>
+              : <div
+                  className="topics-show__title-container flex items-start justify-between">
+                  <h3
+                    ariaLabel="Topic Title"
+                    className="leading-snug lg:pl-14 text-lg lg:text-2xl">
+                    {state.topic |> Topic.title |> str}
+                  </h3>
+                  {isCoach || currentUserId == (firstPost |> Post.creatorId)
+                     ? <button
+                         onClick={_ => send(ShowTopicEditor(true))}
+                         className="topics-show__title-edit-button inline-flex items-center font-semibold p-2 md:py-1 bg-gray-100 hover:bg-gray-300 border rounded text-xs flex-shrink-0 mt-2 ml-4 lg:invisible">
+                         <i className="far fa-edit" />
+                         <span className="hidden md:inline-block ml-1">
+                           {"Edit Title" |> str}
+                         </span>
+                       </button>
+                     : React.null}
+                </div>}
+           <TopicsShow__PostShow
+             key={Post.id(state.firstPost)}
+             post={state.firstPost}
+             topic
+             users
+             posts={state.replies}
+             currentUserId
+             isCoach
+             isTopicCreator={isTopicCreator(firstPost, currentUserId)}
+             updatePostCB={updateFirstPost(send)}
+             addNewReplyCB={addNewReply(send, None)}
+             addPostLikeCB={addFirstPostLike(send)}
+             removePostLikeCB={removeFirstPostLike(send)}
+             markPostAsSolutionCB={() => ()}
+             archivePostCB={() => archiveTopic(communityId)}
+           />
+         </div>}
+        {<h5 className="pt-4 pb-2 lg:ml-14 border-b">
+           {Inflector.pluralize(
+              "Reply",
+              ~count=Array.length(state.replies),
+              ~inclusive=true,
+              (),
+            )
+            |> str}
```

Comment:
Just a note: I've added an `Inflector` util and added a library to help with pluralization in 646c89399b3ba3715a76770ad52d01d649d4eac7.